source/vstd_build/src/main.rs

```diff
@@ -95,6 +95,11 @@ fn main() {
         "--out-dir".to_string(),
         verus_target_path.to_str().expect("invalid path").to_string(),
     ];
+    if let Ok(sysroot) = std::env::var("RUST_SYSROOT") {
+        child_args.push("--sysroot".to_string());
+        child_args.push(sysroot);
+    }
```

Comment:
Why do you need this? Is it to set a sysroot manually, that would otherwise be set by `rustup`?

Reply:
(Somehow missed the notification, my apologies)

Yes, `rustup` wrappers automatically [set the correct `LD_LIBRARY_PATH`](https://github.com/rust-lang/rustup/blob/5af4bc4a0d4bc69ea9091a7935fb3783c5fb508e/src/toolchain/toolchain.rs#L117-L118) (or equivalents on other platforms) when executing the toolchain binaries, and binaries distributed by `rust-overlay` and `fenix` [have their runpaths patched](https://github.com/oxalica/rust-overlay/blob/58240e1ac627cef3ea30c7732fedfb4f51afd8e7/mk-aggregated.nix#L49-L54) so the default sysroot detection logic works.

Reply:
Thanks! Do you know if there's a way to set the sysroot for rustc with an environment variable? Or is it just the `--sysroot` flag?

Reply:
(Sorry again)

I quickly looked, and I couldn't find an environment variable to do this override.